hugo.toml

```diff
@@ -142,9 +142,9 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.29"
+latest = "v1.28"
 
-version = "v1.29"
+version = "v1.28"
```

Comment:
Why this switch from version v1.29 to v1.28?

Reply:
Mistake from previous PR.  Fixed.